src/components/Quiz/Quiz.test.tsx

```
@@ -1339,4 +1384,794 @@
+      expect(titleElement).toBeInTheDocument();
+    });
+  });
+
+  describe('QuizResultPerformance', () => {
+    it('should render performance section with correct statistics', () => {
+      // Mock with questions that have answerKey set
+      const mockQuestionsWithAnswers = [
+        { ...mockQuestion1, answerKey: 'opt1' }, // Correct answer
+        { ...mockQuestion2, answerKey: null }, // No answer
+      ];
+      
```

ESLint

[error] Delete `······`

(prettier/prettier)